### PR TITLE
0.9.0 pre release

### PR DESCRIPTION
**Pull request by stevenhorsman**
The go mod changes look a bit funny as we are going from an alpha release to a pre-release

### CODE REVIEW

File: src/cloud-api-adaptor/go.mod [hunk 1]
```diff
@@ -47,8 +47,8 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/eks v1.29.5
 	github.com/aws/aws-sdk-go-v2/service/iam v1.22.5
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.38.5
-	github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers v0.9.0-alpha1
-	github.com/confidential-containers/cloud-api-adaptor/src/peerpod-ctrl v0.9.0-alpha1
+	github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers v0.9.0-alpha.1
```
Comment (bpradipt) [inline src/cloud-api-adaptor/go.mod:50]: do you want to name the tag as alpha.2 ?

Comment (stevenhorsman) [reply src/cloud-api-adaptor/go.mod:50]: We could do, but it will restart all the e2e jobs again 😞

Comment (bpradipt) [reply src/cloud-api-adaptor/go.mod]: Oh. leave it..

Comment (bpradipt) [reply src/cloud-api-adaptor/go.mod]: Ah I see you already made the change..

**Review by bpradipt — APPROVED** [review]
/lgtm

**Review by wainersm — APPROVED** [review]
Thanks @stevenhorsman !

**Review by beraldoleal — APPROVED** [review]
LGTM, thanks.

**Pull request merged**